React/Views/RCTSliderManager.m

```diff
@@ -76,6 +76,7 @@ - (void)sliderTouchEnd:(RCTSlider *)sender
 
 RCT_EXPORT_VIEW_PROPERTY(value, float);
 RCT_EXPORT_VIEW_PROPERTY(step, float);
+RCT_EXPORT_VIEW_PROPERTY(trackImage, NSString);
```

Comment:
Set the type here to `UIImage` instead of `NSString`, then it will be converted automatically.